demos/python_demos/action_recognition/action_recognition.py

```diff
@@ -41,7 +41,9 @@ def build_argparser():
     args.add_argument("-m_en", "--m_encoder", help="Required. Path to encoder model", required=True, type=str)
     args.add_argument("-m_de", "--m_decoder", help="Required. Path to decoder model", required=True, type=str)
     args.add_argument("-i", "--input",
-                      help="Required. Path to a video or a .txt file with a list of video files (one video per line)", type=str)
+                      help="Required. Id of the video capturing device to open (to open default camera just pass 0), "
```

Comment:
If it's required, you should add `required=True`.

Reply:
Added

Reply:
This also changes the `--help` output, so update the README correspondingly.

Reply:
The output did not change for me.

Reply:
Are you sure? I just tried it myself, and the brackets around `-i INPUT` disappear.

Reply:
Got it. Updated